REFERENCE.md

```diff
@@ -709,11 +711,17 @@ A compatible vehicle is a vehicle that:
 
 #### Return
 
-| Value                      | Type                   | Description                                                                           |
-|:---------------------------|:-----------------------|:--------------------------------------------------------------------------------------|
-| `Compatibility`            | typing.NamedTuple      | The returned object with vehicle's compatibility with the permissions (scope) checked |
-| `Compatibility.compatible` | Boolean                | Whether the vehicle is compatible with the permissions                                |
-| `Compatibility.meta`       | collections.namedtuple | Smartcar response headers (`request_id`, `data_age`, and/or `unit_system`)            |
+| Value                                     | Type                    | Availability          | Description                                                                                                                                         |
+|:------------------------------------------|:------------------------|:----------------------|:----------------------------------------------------------------------------------------------------------------------------------------------------|
+| `Compatibility`                           | typing.NamedTuple       | **API v1.0 and v2.0** |The returned object with vehicle's compatibility with the permissions (scope) checked                                                                | 
+| `Compatibility.compatible`                | Boolean                 | **API v2.0 only**     | Whether the vehicle is compatible with the permissions                                                                                              |
```

Comment:
Isn't `Compatibility.compatible` v1.0 as well?

Reply:
Thanks for catching this, fixing it now